habitat-baselines/habitat_baselines/rl/hrl/hl/high_level_policy.py

```diff
@@ -33,6 +33,10 @@ def __init__(
         self._device = None
         self._agent_name = agent_name
 
+    @property
+    def should_load_agent_state(self):
```

Comment:
I think this will give eval problems, it should not always return False.

Reply:
I think this is the better way to do it because the NN HL policy will override this to return True.